project/plugins.sbt

```diff
@@ -1,7 +1,16 @@
 // Copyright: 2017 https://github.com/cakesolutions/sbt-cake/graphs
 // License: http://www.apache.org/licenses/LICENSE-2.0
 
-import CakePluginDependencies._
+import net.cakesolutions.CakePlatformDependencies.SbtDependencies._
```

Comment:
The code isn't quite duplicated - the `project` directory varies between usages.

Given the two contexts in which this code is used, I think I'd like to try and keep the code associated with the build level it applies to.

Changing the destination file name is an interesting idea. However, my main concern there is that doing this may force an individual to fully understand the build chain? I think it might be better to use the same name throughout so as to re-enforce the semantic intent here.